Makefile

```diff
@@ -62,6 +63,20 @@ build: $(BINDIR)/$(BINNAME)
 $(BINDIR)/$(BINNAME): $(SRC)
 	GO111MODULE=on go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME) ./cmd/helm
 
+# ------------------------------------------------------------------------------
+#  install
+
+.PHONY: install
+install:
+	@if [ ! -f "$(BINDIR)/$(BINNAME)" ]; then \
+		echo "please run 'make build' first" && \
+		exit 1; \
+	fi
+
+	@echo "Preparing to install $(BINNAME) into ${HELM_INSTALL_DIR}"
+	@ln -sf "$(BINDIR)/$(BINNAME)" "$(HELM_INSTALL_DIR)/$(BINNAME)"
```

Comment:
A symbolic link should not be used here. If you `rm -rf ..` as most users do after they've compiled the program, helm will no longer be found on the system.

Most `install` targets use `install` instead. Have a look at the [kernel documentation on kbuild](https://www.kernel.org/doc/html/v5.4/kbuild/makefiles.html) on how they configure their `install` target. `INSTALL_PATH` is the common variable for referencing where binaries should be installed, for example.

See `man install` for more information.